<!-- -->
<launch>
    <node pkg="cupoch_conversions" type="pcl_conversions_test_node" name="pcl_conversions_test_node" output="screen">
        <param name="camera3d_point_topic" value="/points_cloud" />
    </node>

</launch>